tests/helpers.py

```diff
@@ -2169,3 +2169,42 @@ def collect_performance_stats():
     file_name = os.path.join(log_dir_path, 'performance')
     with open(file_name, 'w') as outfile:
         json.dump(performance_stats, outfile)
+
+
+def check_local_volume():
+    """
+    Function to check if Local-volume is present or not
+    Returns:
+        bool: True if LV present, False if LV not present
+    """
+    ocp_obj = OCP()
+    command = "get localvolume -n local-storage "
+    status = ocp_obj.exec_oc_cmd(command, out_yaml_format=False)
+    return "No resources found" not in status
+
+
+@retry(AssertionError, 12, 10, 1)
+def check_pvs_created(num_of_pvs):
+    """
+        Verify that exact number of PVs were created and are in the Available state
+
+        Args:
+            num_of_pvs (int): number of pvs required
+
+        Raises:
+            AssertionError: if the PVs are not in the Available state
+
+        """
+    logger.info(f"Verifying atleast {num_of_pvs} PVs are created")
+    out = run_cmd("oc get pv -o json")
+    pv_json = json.loads(out)
+    current_count = 0
+    for pv in pv_json['items']:
+        pv_state = pv['status']['phase']
+        pv_name = pv['metadata']['name']
+        logger.info("%s is %s", pv_name, pv_state)
+        if pv_state == 'Available':
+            current_count = current_count + 1
+    assert current_count >= num_of_pvs, (
+        f"Current Available PV count is {current_count}"
```

Comment:
you can make use of OCP class to get PV and can check the status of it

Reply:
I can but the problem is there are already 3-4 PVs which are bound and new PVs are created which will be in Available state. Hence, this seemed to be the best option.